scripts/install_bmi.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -ex
+sudo apt-get install -y python-pip
+### Packages for BMI
+sudo apt-get install -y tgt sqlite3 virtinst
+
+git clone http://github.com/cci-moc/hil
+
+pushd hil/
+sudo apt-get install -y libvirt-bin bridge-utils ipmitool telnet apache2 libapache2-mod-wsgi python-pip qemu-kvm python-libvirt python-psycopg2 vlan net-tools icu-devtools libicu-dev libxml2-dev libxslt1-dev python-virtualenv python3-virtualenv virtualenv # For Ubuntu
+
```

Comment:
Maybe we could do `cat /etc/*-release` to get the name of distro and based on it install the required packages. Because we are using CentOS (or RHEL) almost everywhere.